public/stations/index.html

```diff
@@ -2,96 +2,89 @@
-    />
-    <meta
-      name="keywords"
-      content="Chargetrip, Chargetrip API, EV, ev-cars, ev-routing, tile service, clusterbuster, map, filters"
-    />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;600&display=swap"
-      rel="stylesheet"
-    />
+    <meta name="description" content="Query particular station with the Chargetrip API." />
+    <meta name="keywords" content="Chargetrip, Chargetrip API, EV, ev-cars, ev-routing, station" />
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap" rel="stylesheet" />
     <link rel="icon" href="../favicon-57.png" type="image/x-icon" />
     <link href="https://api.tiles.mapbox.com/mapbox-gl-js/v1.4.1/mapbox-gl.css" rel="stylesheet" />
     <link rel="stylesheet" href="style.css" />
   </head>
 
   <body>
     <div id="map"></div>
-    <div class="github-link">
-      <a href="https://github.com/chargetrip/examples/tree/master/stations" title="github project">
-        View on Github
-      </a>
-      <button aria-label="remove github link" title="remove github link" onclick="this.parentElement.remove()"></button>
-    </div>
-
-    <button id="centerMe" class="centerMe-button">📌</button>
-
-    <div class="station-filters">
+    <div class="card">
       <header>
-        <div>Showing <span id="stationAmount"></span> stations</div>
-        <div class="subtext">Note: example is limited to 20 stations</div>
+        <h1>13 stations</h1>
+        <p>Limited to 20 stations</p>
       </header>
-      <div class="section">
-        <h2>Reachable from your location</h2>
-        <div class="range-slider">
-          <label class="range-thumb" id="range-thumb" for="range"></label>
-          <input type="range" min="1000" max="10000" value="3000" step="1000" id="range" />
-          <div class="range-info">
-            <span>1</span>
-            <span>10</span>
+      <main>
+        <section>
+          <div class="range-slider">
+            <div class="labels">
+              <label>Distance from location</label>
+              <label id="distance" for="range"></label>
+            </div>
+            <input type="range" min="1000" max="10000" value="3000" step="1000" id="range" />
           </div>
+        </section>
+        <section>
+          <h3>Charging power</h3>
+          <ul class="checklist power">
+            <li>
+              <input type="checkbox" class="filter-input" id="turbo" value="turbo" />
+              <label for="turbo">
+                <span class="title">Turbo - 100 to 350 kW</span>
+                <span class="subtitle">Aprox. 30 min charge</span>
+              </label>
+            </li>
+            <li>
+              <input type="checkbox" class="filter-input" id="fast" value="fast" />
+              <label for="fast">
+                <span class="title">Turbo - 100 to 350 kW</span>
```

Comment:
I see 3 times Turbo, this should be 
Slow until 43 kWh Fast 43 till 100 kWh and Turbo 100 to 350 kWh